CONTRIBUTING.md

```diff
@@ -49,134 +37,181 @@ and let us know if it's not up-to-date (even better, submit a PR with your  corr
 
 ## Getting Started
 
-### Gitpod
+The following steps describe how to set up the AWS CDK repository on your local machine.
+The alternative is to use [Gitpod](https://www.gitpod.io/), a Cloud IDE for your development.
+See [Gitpod section](#gitpod) on how to set up the CDK repo on Gitpod.
 
-For setting up a local development environment,
-we recommend using [Gitpod](http://gitpod.io) -
-a service that allows you to spin up an in-browser
-Visual Studio Code-compatible editor,
-with everything set up and ready to go for CDK development.
-Just click the button below to create your private workspace:
-
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/aws/aws-cdk)
-
-This will start a new Gitpod workspace,
-and immediately kick off a build of the CDK code.
-Once it's done (it takes around an hour, unfortunately),
-you can work on any package that you want to modify,
-as described in ['Quick Iteration'](#quick-iteration) below.
-
-Gitpod is free for 50 hours per month -
-make sure to stop your workspace when you're done
-(you can always resume it later, and it won't need to run the build again).
+### Setup
 
-### Local dependencies
-
-If you don't want to use Gitpod,
-you need to have the following SDKs and tools locally:
+The following tools need to be installed on your system prior to installing the CDK:
 
 - [Node.js >= 10.13.0](https://nodejs.org/download/release/latest-v10.x/)
   - We recommend using a version in [Active LTS](https://nodejs.org/en/about/releases/)
   - ⚠️ versions `13.0.0` to `13.6.0` are not supported due to compatibility issues with our dependencies.
 - [Yarn >= 1.19.1, < 2](https://yarnpkg.com/lang/en/docs/install)
-- [Java >= OpenJDK 8, 11, 14](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html)
-- [Apache Maven >= 3.6.0, < 4.0](http://maven.apache.org/install.html)
-- [.NET Core SDK 3.1.x](https://www.microsoft.com/net/download)
```

Comment:
Unfortunately, .NET Core is required (to run the test cases inside the `aws-cdk` package).